docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+# Creating a Bridge With a Dead Letter Sink (DLS)
+
+## What is a Dead Letter Sink?
+
+Knative Eventing provides various configuration parameters to control the delivery of events in case of failure. For instance, you can decide to retry sending events that failed to be consumed, and if this didn't work you can decide to forward those events to a dead letter sink.
+
+## Implementing a Dead Letter Sink
+
+There are two ways to implement a dead letter sink, either by use of a Subscription or by use of a Broker.
+
+### Subscription Configuration
+
+You can configure how events are delivered for each Subscription by adding a delivery spec to the Subscription object, as shown in the following example:
+
+```yaml
+apiVersion: messaging.knative.dev/v1
+kind: Subscription
+metadata:
+  name: example-subscription
+  namespace: example-namespace
+spec:
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
+### Broker Configuration
+
+You can configure how events are delivered for each Broker by adding a delivery spec, as shown in the following example:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
```

Comment:
I should know the answer to this, but does using kind: Broker require any additional installation beyond the main kubectl apply commands? For instance does it require installing Trigger Flow?

Reply:
we can see here the API version is `eventing.knative.dev/v1` so as long as the Knative eventing API/CRD's are available, one can deploy a broker. Triggerflow is not a dependent here

Reply:
Hey Jeff, i'm really sorry but I must have been a bit confused and tired when I made that comment. I meant to add it to the "kind:Bridge" further down. That is the piece that is not part of the OSS product therefore we can't have it in these guides (unless i'm mistaken). Do you think we can rewrite it as a single yaml file but with separate components (no kind:bridge)?